OrbitQt/topdownwidget.cpp

```diff
@@ -32,6 +34,7 @@ const QString TopDownWidget::kActionCollapseChildrenRecursively =
 const QString TopDownWidget::kActionExpandAll = QStringLiteral("Expand all");
 const QString TopDownWidget::kActionCollapseAll = QStringLiteral("Collapse all");
 const QString TopDownWidget::kActionLoadSymbols = QStringLiteral("&Load Symbols");
+const QString TopDownWidget::kActionDisassembly = QStringLiteral("Go to &Disassembly");
```

Comment:
Why is the "&" there? Is this a special purpose char, or is this by accident.

Reply:
It enables "D" as a shortcut, also see the other actions.

Reply:
Thanks for clarifying